test/unit-test/core_mqtt_serializer_utest.c

```diff
@@ -1660,10 +1654,10 @@ void test_MQTT_DeserializePublish( void )
 
     /* Create a PUBLISH packet to test. */
     memset( &publishInfo, 0x00, sizeof( publishInfo ) );
-    publishInfo.pTopicName = "/test/topic";
-    publishInfo.topicNameLength = ( uint16_t ) strlen( publishInfo.pTopicName );
-    publishInfo.pPayload = "Hello World";
-    publishInfo.payloadLength = ( uint16_t ) strlen( publishInfo.pPayload );
+    publishInfo.pTopicName = TEST_TOPIC_NAME;
+    publishInfo.topicNameLength = TEST_TOPIC_NAME_LENGTH;
+    publishInfo.pPayload = MQTT_SAMPLE_PAYLOAD;
+    publishInfo.payloadLength = MQTT_SAMPLE_PAYLOAD_LEN;
```

Comment:
Could you possible use `setupWillInfo` for these instead, changing what is necessary for the specific test case? Maybe using the built in unity fixture, `setUp`, can make this easier for you.

Reply:
Sure, I'll rename it to `setupPublishInfo`